src/implementations/twig/components/site-header/eu-site-header.story.js

```diff
@@ -65,75 +66,139 @@ const getArgTypes = (data) => {
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the login box for a logged in user',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
   }
   argTypes.show_language_selector = {
     name: 'language selector',
-    type: { name: 'boolean' },
+    control: { type: 'boolean' },
     description: 'Show the language selector',
     table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: 'true' },
       category: 'Optional',
     },
+    mapping: {
+      0: false,
+      1: true,
+    },
   };
   argTypes.show_search = {
     name: 'search (Europa Search)',
-    type: { name: 'boolean' },
+    control: { type: 'boolean' },
     description: 'Show the search form',
     table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: 'true' },
       category: 'Optional',
     },
+    mapping: {
+      0: false,
+      1: true,
+    },
   };
   if (data.notification) {
     argTypes.show_notification = {
       name: 'notification',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the notification box',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
+    };
+  }
+  if (data.basket) {
+    argTypes.show_basket = {
+      name: 'basket',
+      control: { type: 'boolean' },
+      description: 'Show the basket',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
+        category: 'Optional',
+      },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
   }
   if (data.has_menu) {
     argTypes.show_menu = {
       name: 'menu',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the menu',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'true' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
   }
   if (data.cta_link) {
     argTypes.show_cta_link = {
       name: 'call to action',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the call to action link',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
   }
   if (data.site_name) {
     argTypes.show_site_name = {
       name: 'site name',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the site name',
       table: {
+        type: { summary: 'boolean' },
```

Comment:
mmh..no, i think this is confusing, we don't have a parameter show_site_name, right? this is just a toggle we use in storybook, so we shouldn't define a "table" for it